include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -78,6 +78,11 @@ static const char* const kOrtSessionOptionsConfigAllowIntraOpSpinning = "session
 // has to guarantee that the model bytes are valid until the ORT session using the model bytes is destroyed.
 static const char* const kOrtSessionOptionsConfigUseORTModelBytesDirectly = "session.use_ort_model_bytes_directly";
 
+// It controls whether qdq is int8 allowed in QDQ format model or not.
```

Comment:
"whether qdq is int8 allowed in QDQ format model"
I didn't quite understand that.
maybe it would be good to mention that it should be specified for the target hardware.